code/modules/surgery/surgery_step.dm

```diff
@@ -175,3 +175,16 @@
 	if(!target_detailed)
 		var/you_feel = pick("a brief pain", "your body tense up", "an unnerving sensation")
 		target.show_message(vague_message, MSG_VISUAL, span_notice("You feel [you_feel] as you are operated on."))
+/**
+ * Sends a pain message to the target, including a chance of screaming.
+ *
+ * Arguments:
+ * * target - Who the message will be sent to
+ * * pain_message - The message to be displayed
+ * * mechanical_surgery - Boolean flag that represents if a surgery step is done on a mechanical limb (therefore does not force scream)
+ */
+/datum/surgery_step/proc/display_pain(mob/living/target, pain_message, mechanical_surgery = FALSE)
```

Comment:
you might wanna add a parameter to this that determines whether or not the pain message will have a chance to produce a scream (or just make the parameter directly determine the chance for a scream), since some of these pain messages don't really seem scream-worthy

Reply:
Other than the mechanical limb surgeries, which are already not scream causing, which ones do you mean?